packages/url_launcher/url_launcher_web/test/pubspec.yaml

```diff
@@ -14,9 +15,9 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   http: ^0.12.2
```

Comment:
This needs to be moved to the null-safe version. Why isn't anything complaining? Unused import?

Reply:
Cleaned up